src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-router.ts

```diff
@@ -46,8 +50,8 @@ class ZWaveJSConfigRouter extends HassRouterPage {
     el.isWide = this.isWide;
     el.narrow = this.narrow;
     el.configEntryId = this._configEntry;
-    if (this._currentPage === "node") {
-      el.nodeId = this.routeTail.path.substr(1);
+    if (this._currentPage === "node_config") {
+      el.deviceId = this.routeTail.path.substr(1);
```

Comment:
Don't do this. Instead, `el.route` is already set to the routeTail so the element can extract it themselves.